project/test/resources/OPConsultantRecord-Output.json

```diff
@@ -1,2688 +1,2143 @@
+        }, {
+          "reference": "urn:uuid:40f948cd-8c5d-497b-85b7-eb2f24a0faa9",
+          "display": "Condition"
+        } ]
+      }, {
+        "title": "Allergies",
+        "code": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "722446000",
+            "display": "Allergy record"
+          } ]
+        },
+        "entry": [ {
+          "reference": "urn:uuid:a5407f45-8867-47d1-aeea-3523fbafbaeb",
+          "display": "AllergyIntolerance"
+        }, {
+          "reference": "urn:uuid:45060018-0468-4418-87a9-9123b527f863",
+          "display": "AllergyIntolerance"
+        } ]
+      }, {
+        "title": "Medical History",
+        "code": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "371529009",
+            "display": "History and physical report"
+          } ],
+          "text": "History and physical report"
+        },
+        "entry": [ {
+          "reference": "urn:uuid:ebebf976-248e-482e-9045-d048fd0abbf4",
+          "display": "Condition"
+        }, {
+          "reference": "urn:uuid:56fa897d-624c-40cb-a319-d1aaabc9d7e7",
+          "display": "Condition"
+        }, {
+          "reference": "urn:uuid:df2b8004-2a2c-4eab-90e4-42e04f2cd7ca",
+          "display": "Condition"
+        }, {
+          "reference": "urn:uuid:fd9f235d-0098-4c0f-87b8-62cb73bf615f",
+          "display": "Condition"
+        }, {
+          "reference": "urn:uuid:ad3a48ff-ae0f-434a-8b15-15a5c74a63e8",
+          "display": "Procedure"
+        }, {
+          "reference": "urn:uuid:7afd56b8-c445-4500-b1ea-66edf92e360e",
+          "display": "Procedure"
+        } ]
+      }, {
+        "title": "Adverse Events",
+        "code": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "405533003",
+            "display": "Adverse incident outcome categories"
+          } ],
+          "text": "Adverse Events"
+        },
+        "entry": [ {
+          "reference": "urn:uuid:54ea988c-21d1-4dfb-b58e-2b0cee43d0f2",
+          "display": "AdverseEvent"
+        } ]
+      }, {
+        "title": "OtherObservations",
+        "code": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "364313002",
+            "display": "Measure of menstruation"
+          } ],
+          "text": "Menstruation History"
+        },
+        "entry": [ {
+          "reference": "urn:uuid:aec6c97c-ef8c-41bb-b025-81279cfee32a",
+          "display": "Observation"
+        }, {
+          "reference": "urn:uuid:e2b4631a-2b86-40b9-a943-2fee7e1da5c7",
+          "display": "Observation"
+        } ]
+      }, {
+        "title": "InvestigationAdvice",
+        "code": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "721963009",
+            "display": "Order document"
+          } ],
+          "text": "Examination Details"
```

Comment:
its not examination details - yet. Better to say something like "Investigation Advice"